### ISSUE TITLE
Update duration more frequently for running jobs

### ISSUE THREAD

**Issue by mmurto** [eclipse-apoapsis/ort-server#1079]
The duration of jobs while running in `/organizations/${orgId}/products/${productId}/repositories/${repoId}` stay frozen for 10 seconds, while showing the exact time up to the second. This can feel like it has already ended, so the duration should either update every second, or show the duration in less detail, like "< 1 minute".

**Comment by mnonnenmacher** [comment]
I like seeing the seconds, I think this could be implemented with a timer on the client.

**Comment by lamppu** [comment]
The refetch interval is ten seconds, so do you think it would be a problem if the duration would update every second and then when the data is refetched, and a run would be finished, that there could be this jump from a higher duration to the actual final duration?

**Comment by mnonnenmacher** [comment]
For me, that would be fine.

**Comment by mmurto** [comment]
> The refetch interval is ten seconds, so do you think it would be a problem if the duration would update every second and then when the data is refetched, and a run would be finished, that there could be this jump from a higher duration to the actual final duration?

Yep, that's the way to go IMO. Update client side time at least once a second, and then use correct one after refetch if it's finished.

**Comment by Etsija** [comment]
@lamppu with #1265, can this be closed?

**Comment by lamppu** [comment]
> @lamppu with #1265, can this be closed?

Yes
